packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -16,6 +17,7 @@ const displayName = params.displayName;
 const token = params.token;
 const groupId = params.groupId;
 const userId = params.userId;
+const locale = params.locale;
```

Comment:
nit: This is a test so easy to change. In general, keep the API as tight as possible. Here, the only supported value is really `fr-FR`. This is not going to be obvious from the test code that sets this parameter.

Would be preferable to make this param `params.useFrLocale`, since that's all you need for the test anyway. If later we wanted to add more locales, we can always change this to be more generic.